python/lsst/ts/m2/__init__.py

```diff
@@ -19,6 +19,15 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+# Set __version__ before importing the CSC
+try:
```

Comment:
Please, remove the "try/except" import of `version` in lines 34-37.

Reply:
Done in the commit: baf6155. Thanks!